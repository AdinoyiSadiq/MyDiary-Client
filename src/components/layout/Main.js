import React from 'react';
import Header from '../../containers/Header';

const Main = (props) => {
  const { children } = props;
  return (
    <div>
      <Header {...props}/>
      {children} 
    </div>
  );
};

export default Main;