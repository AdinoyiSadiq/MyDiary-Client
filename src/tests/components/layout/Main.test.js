import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Main from '../../../components/layout/Main';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<Main />);
});


describe('Main UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});