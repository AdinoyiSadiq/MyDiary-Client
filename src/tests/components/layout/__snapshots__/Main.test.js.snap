// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main UI render features view should render as expected 1`] = `
<div>
  <Header />
</div>
`;
